ci-operator/step-registry/openshift/manifests/crun/openshift-manifests-crun-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_mc-master-crun.yml" << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-crun
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KZGVmYXVsdF9ydW50aW1lID0gImNydW4iCltjcmlvLnJ1bnRpbWUucnVudGltZXMuY3J1bl0KcnVudGltZV9yb290ID0gIi9ydW4vY3J1biIK
```

Comment:
nit: probably easier to maintain if you write this to a `/tmp` file and then use `base64` to encode it.  Something like:

```sh
cat <<EOF >/tmp/50-crun
[crio.runtime]
default_runtime = "crun"
[crio.runtime.runtimes.crun]
runtime_root = "/run/crun"
EOF

cat <<EOF > "${SHARED_DIR}/manifest_mc-master-crun.yml"
apiVersion: machineconfiguration.openshift.io/v1
...
          source: data:text/plain;charset=utf-8;base64,$(base64 -w0 </tmp/50-crun)
...
EOF
```